fix: moved alerts and reports default values to config

Always-prog opened the pull request:
<!---
Please write the PR title following the conventions at https://www.conventionalcommits.org/en/v1.0.0/
Example:
fix(dashboard): load charts correctly
-->

### SUMMARY
Typically, users do not verify the working timeout and default repetition for creating alerts or reports. As a result, if a Superset administrator wants to change the default values for these options, they need to modify the code. To address this issue, I submitted a pull request that moves these default values to the config file.


### ADDITIONAL INFORMATION
<!--- Check any relevant boxes with "x" -->
<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

Always-prog committed:
feat(alerts reports): moved default alert vals to config

Always-prog committed:
build fixes

Always-prog committed:
fix(alerts reports): Added ALERT_REPORTS_ to default vals

villebro commented on superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx:
```diff
@@ -512,6 +495,28 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
+    ALERT_REPORTS_DEFAULT_RETENTION,
+  } = useSelector<any, AlertsReportsConfig>(state => state.common.conf);
+
+  const DEFAULT_ALERT = {
```
nit: very minor detail, but as this is not technically a constant, it might be be worth considering renaming this to `defaultAlert`.

Always-prog replied:
Thanks for soon reviewing!
Renamed.

villebro reviewed (approved):
Nice improvement 👍 LGTM and tested to work as expected with one minor naming comment.

Always-prog committed:
fix(alerts reports): renamed defaultAlert because it isn't technically constant anymore

Always-prog committed:
fix(alerts reports): reformat base.py front config keys

Always-prog committed:
fix(alerts reports): Default conf values for passing test cases

Always-prog committed:
fix(alerts reports): moved default values to default config.

Always-prog committed:
fix(alerts reports): fixed wrong default values

Always-prog committed:
fix(alerts reports): use default value it not defined in conf

Always-prog commented:
@villebro Hello!
I fixed the passing test cases because the test cases were running without the python config. We still need to have default values in the code. However, if they are declared in config, they will be used. 
Can you check my code?

villebro commented:
> @villebro Hello!
> I fixed the passing test cases because the test cases were running without the python config. We still need to have default values in the code. However, if they are declared in config, they will be used.
> Can you check my code?

@Always-prog yeah, this is a bit of a problem when the frontend assets don't have context on regarding the python config values. You can probably either mock the values on the test (more work) or just add the same defaults on the frontend code path (less work). I'm ok with both.

Always-prog commented:
> > @villebro Hello!
> > I fixed the passing test cases because the test cases were running without the python config. We still need to have default values in the code. However, if they are declared in config, they will be used.
> > Can you check my code?
> 
> @Always-prog yeah, this is a bit of a problem when the frontend assets don't have context on regarding the python config values. You can probably either mock the values on the test (more work) or just add the same defaults on the frontend code path (less work). I'm ok with both, as in practice they won't affect this functionality.

I added default values in code.

villebro commented:
@Always-prog can you rebase the PR? That might help resolve that failed test.

Always-prog commented:
Hello @villebro and @rusackas!
I have fixed the frontend tests and they are now passing. Could you please review my code once more?

Always-prog commented:
I need to create a new PR for the fix?
There are many commits that are not mine, due to rebase.

dpgaspar reviewed (dismissed):
probably something went wrong with the rebase

Always-prog committed:
fix(alerts reports): python config can be undefined

Always-prog commented:
> probably something went wrong with the rebase

I reverted commit with rebasing.

Always-prog commented:
On my fork all CI test has passed.
![image](https://user-images.githubusercontent.com/66589759/217454455-5685e6c6-11fb-412b-b608-78c250df98cc.png)

Maybe need to restart them @dpgaspar?

villebro commented:
@dpgaspar I believe the rebasing problems have been resolved, can you take another look?

Always-prog commented:
@dpgaspar Hello!
Can you please review py PR again?

villebro reviewed (approved):
LGTM, thanks for this and apologies for letting this get stale

Always-prog commented:
Thanks for approving!
